src/destination/tpl/selector.tpl

```diff
@@ -4,9 +4,18 @@
     {{/if}}
     <div>
         {{#if description}}
-        <p>{{description}}</p>
+        <p class="section-title">{{description}}</p>
         {{/if}}
         <div class="selector-container"></div>
+        {{#if showACL}}
+        <div class="permissions-settings-container">
+            <p class="section-title">{{__ 'Permission settings'}}</p>
+            <input type="radio" class="acl-mode-radio acl-keep-original" {{#equal aclTransferMode 'acl.keep.original' }}checked {{/equal}} value="acl.keep.original" />
+            <label for="acl-keep-original">{{__ 'Keep the original permissions'}}</label><br>
+            <input type="radio" class="acl-mode-radio acl-use-destination" {{#equal aclTransferMode 'acl.use.destination' }}checked {{/equal}} value="acl.use.destination" />
+            <label for="acl-use-destination">{{__ 'Inherit the permission of the parent folder'}}</label>
```

Comment:
**issue:** The label is not linked with the input. Actually, the `for` attribute needs an ID.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

**suggestion:** Use dynamic ID generated from the component
```suggestion
            <input id="acl-keep-original{{uniqId}}" type="radio" class="acl-mode-radio acl-keep-original" {{#equal aclTransferMode 'acl.keep.original' }}checked {{/equal}} value="acl.keep.original" />
            <label for="acl-keep-original{{uniqId}}">{{__ 'Keep the original permissions'}}</label><br>
            <input id="acl-use-destination{{uniqId}}" type="radio" class="acl-mode-radio acl-use-destination" {{#equal aclTransferMode 'acl.use.destination' }}checked {{/equal}} value="acl.use.destination" />
            <label for="acl-use-destination{{uniqId}}">{{__ 'Inherit the permission of the parent folder'}}</label>
```